tests/e2e/api/account.test.ts

```diff
@@ -1,10 +1,9 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-import { Aptos, AptosConfig, Network } from "../../../src";
-
-// TODO
-// add account getTransactions tests once sdk v2 supports faucet (which needs transaction operation support)
+import { Account, Aptos, AptosConfig, Network } from "../../../src";
+import { U64 } from "../../../src/bcs/serializable/move-primitives";
```

Comment:
With the exports this should all collapse!  Hopefully everything will be easily accessible from one path.